README.rst

```diff
@@ -328,6 +331,24 @@ Possible use cases are the following:
     >>> p.name
     'pi+'
 
+    >>> cid = Corsika7ID(5)
+    >>> p = Particle.from_pdgid(cid.to_pdgid())
+    >>> p.name
+    'mu+'
+
+Corsika7
```

Comment:
I see that you even add a little section in the README for Corsika7. We do not have it for others but I see it is a good thing. You're telling me we could/should expand this doc section on converters (at least while we do not have Sphinx docs). OK, cool.